client/core/types.go

```diff
@@ -650,6 +660,10 @@ func (a *dexAccount) setupCryptoV2(creds *db.PrimaryCredentials, crypter encrypt
 	for i := 0; i < 8; i++ {
 		kids = append(kids, binary.LittleEndian.Uint32(byteSeq[i*4:i*4+4]))
 	}
+	// Last child is the account key index, which is 0 for the primary/default
+	// account and 0 > k < 2^31 for supplementary accounts where previously
+	// derived account keys have been cleared/reset.
+	kids = append(kids, keyIndex)
```

Comment:
I know we haven't released an HD keys dex client yet, but anyone that was using HD accounts prior to this commit will then loose the ability to restore their previous account because we now starting one branch deeper.  Is that correct?

Reply:
It is. Can be resolved by not appending last child if `keyIndex == 0` but don't know if that's necessary since there's no live client with this feature yet.

Reply:
Just testnet accounts is all.  Devs can re-register on dex-test if they did use it with the prior derivation path.